pkg/syncer/resource_syncer.go

```diff
@@ -404,16 +404,13 @@ func (r *resourceSyncer) processNextWorkItem(key, name, ns string) (bool, error)
 
 	klog.V(log.LIBTRACE).Infof("Syncer %q retrieved %sd resource %q: %#v", r.config.Name, op, resource.GetName(), resource)
```

Comment:
Is %sd a typo?

Reply:
No - it's purposely appends a 'd' to op name.